nodes/griptape_nodes_library/drivers/web_search/web_search_drivers.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, name: str, metadata: dict | None = None) -> None:
 
         self.add_parameter(
             Parameter(
-                name="driver",
+                name="web_search_driver",
```

Comment:
leaving an FYI - changed this in order to clarify it's a web_search_driver, not another kind of driver